components/IndicatorsCmip6ChartControls.vue

```diff
@@ -27,8 +30,6 @@ chartStore.labels = {
     'KACE-1-0-G': 'KACE-1-0-G',
     MIROC6: 'MIROC6',
     'MPI-ESM1-2-LR': 'MPI-ESM1-2-LR',
-    'NorESM2-MM': 'NorESM2-MM',
```

Comment:
Hmm, why is this being removed?  Is it "not ready yet" or are our docs out of sync a bit in terms of what is in/out of the regridded stuff?

Reply:
`NorESM2-MM` no longer appears in the CMIP6 indicators output (example: http://development.earthmaps.io/indicators/cmip6/point/61.5/-147), but I'm not sure why. I've removed it here to keep the webapp in sync with the current development API.